### DISCUSSION TITLE
Reading Lists - Add options to exclude auto creation

### DISCUSSION THREAD

**Discussion by kingfenix** [gotson/komga#1529] [Ideas]
### Describe your suggested feature

I would like a couple settings for reading lists to appear possibly in server settings. 

1. Minimum Issues required, would take a numerical input and defaulting to "1" would act as it currently does. But when set to "2" would eliminate creating reading lists with only 1 book which is pointless.
2. Minimum Series required, again would take a numerical input and default to "1" would be current behavior. Set to "2" would require more than a single series to create the reading list.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by kingfenix** [comment]
an alternative would be to "save" and not recreate read lists that are deleted manually. When I delete reading lists they just come back....

**Comment by gotson** [comment]
Probably easier to implement #1765
